website/src/components/apply/apply.tsx

```diff
@@ -35,6 +35,19 @@ function Component() {
 }
 
 function sendMessage() {
+  if (
+    form.firstName.current.value.trim() == "" &&
+    form.lastName.current.value.trim() == "" &&
+    form.email.current.value.trim() == "" &&
+    form.phone.current.value.trim() == "" &&
+    form.linkedin.current.value.trim() == "" &&
+    form.github.current.value.trim() == "" &&
+    form.blurb.current.value.trim() == ""
+  ) {
+    alert("Please fill out the form before submitting");
```

Comment:
Alert!? Idk if I'd want to work at a place that uses alert() unironically

Reply:
this is ironic usage

Reply:
:(

Reply:
If you want to add a TODO for me to use an alert service type thing to show a butter bar or something like we do in the app, that's cool with me.

Reply:
looking forward to your beautiful alert-thats-not-an-alert